### ISSUE TITLE
PDF Library using characters from non english rise error 

### ISSUE THREAD

**Issue by IlfirinPL** [robocorp/rpaframework#304]
When using non english(using UTF-8 encoding) characters to generate pdf, function raise error UnicodeEncodeError: 'latin-1

Code to reproduce 
```
    Create File    Template.html    {{name}}
    ${payload}    Create Dictionary    name=ĄĆĘŁŃÓŚŹŻąćęłńóśźż
    Template Html To Pdf    Template.html    output.pdf    variables=${payload}
```

Error 
UnicodeEncodeError: 'latin-1' codec can't encode characters in position 0-4: ordinal not in range(256)

**Comment by mikahanninen** [comment]
Thank you for the report! We will investigate and fix the issue.

**Comment by cmin764** [comment]
@IlfirinPL This got fixed now with **rpaframework-pdf** `1.30.0`. Add this version in your _conda.yaml_ (`rpaframework-pdf==1.30.0`) or just rebuild your env to get the latest update.

**Comment by IlfirinPL** [comment]
@cmin764  Thank you, It`s working perfectly
